File: Mod07/Democode/VB/PolymorphismDemo_VB/PolymorphismDemo_VB/Student.vb
Public Class Student
    Inherits Person

    Private _school As String
    Private _program As String

    Public Property School As String
        Get
            Return Me._school
        End Get
        Set(value As String)
            Me._school = value
        End Set
    End Property

    Public Property Program As String
        Get
            Return Me._program
        End Get
        Set(value As String)
            Me._program = value
        End Set
    End Property

    Public Sub sleep()

    End Sub

    Public Sub study()

    End Sub

    Public Overrides Sub move()
        Console.WriteLine("Moving")
    End Sub


End Class

File: Mod07/Democode/CS/PolymorphismDemo_CS/PolymorphismDemo_CS/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolymorphismDemo_CS
{
    class Student : Person
    {
        private string school;
        private string program;

        public string School
        {
            get { return school; }
            set { school = value; }
        }
        
        public string Program
        {
            get { return program; }
            set { program = value; }
        }

        public void sleep()
        {

        }

        public void study()
        {

        }

        public override void move()
        {
            Console.WriteLine("Moving");
        }

       
    }
}
